plugins/Morpheus/stylesheets/general/_forms.less

```diff
@@ -83,6 +82,11 @@ select {
             font-size: inherit;
         }
     }
+
+    .checkbox {
+      font-size: 14px;
+    }
```

Comment:
please change the original style rather than adding yet another selector… that would also avoid to forget styling `.radio`.